lib/sles4sap/ipaddr2.pm

```diff
@@ -954,7 +983,7 @@ sub ipaddr2_cloudinit_sanity {
     foreach my $id (1 .. 2) {
         foreach (
             'grep -E "root|cloudadmin|hacluster" /etc/passwd',
-            'zypper se -s -i cloud-init',
+            'sudo zypper se -s -i cloud-init',
```

Comment:
I do not think sudo is needed for `se` `search`

Reply:
After doing zypper patch, when running 'zypper se xxx' it will rebuild repository, such as https://openqaworker15.qa.suse.cz/tests/309577#step/cluster_create/18 . This requires `sudo`, otherwise the `zypper se` will fail in `permission issue`

Reply:
I'd propose to run a `zypper ref` after the `zypper patch` and remove sudo from here

Reply:
The first time VR run for `zypper se -s -i`, failed in permission. Then I added the `sudo` for it. But the second run, it failed in another `zypper se -s -i`, still is the permission issue. So I think adding `zypper ref` maybe don't make sense, if it works, the second `zypper se` should not fail in permission issue.  But I am not 100% sure, haven't tested it.